docs/core/integration-quality-scale/rules/iqs044.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "IQS044 - The documentation describes all integration installation parameters"
+---
+
+IQS044 is as follows:
+
+The documentation describes all integration installation parameters
```

Comment:
Isn't this better described with data description parameters in the config flow?

Reply:
Right, but do we really need a special description of all the parameters in the docs. Isn't a general description of the setup flow plus data descriptions in the config flow enough? It feels like we're putting effort in the wrong place here.

Reply:
Yes; in some cases it provides additional context we can't provide in the UI. Additionally, this is a comment/feedback/question that often comes back from end-users.

Reply:
From the documentation perspective, we don't agree on that. We want to provide a consistent user experience over time. Just like we want a consistent user experience in the UI.

Reply:
Putting the description in both places introduces the risk for inconsistencies in the description between the user interface and the documentation.

Reply:
I think that is something we should keep an eye out for then and maybe automate (if we find new config flow strings in the strings.json in a commit, we should add the docs-missing label)

Reply:
That doesn't solve the problem. Until it's completely automatically synced between core and docs, the docs can be incorrect. The risk is high for incorrectness for things that are listed and more or less repetitions where the mind tries to take shortcuts when reading.

Reply:
The risk has always been high, and there is no reason not to do it.

This is not an new issue and honestly, if the requirement for documentation is that we can only document things that cannot go out of sync, we can just as well just delete our documentation. Everything can go out of sync.

Reply:
I think this is different since for most parameters the text could be exactly the same as the text in the core source. Copying it to a different repo manually is just a terrible way of doing things that is error prone, doesn't scale well and will just increase workload without much value for most integrations.

Reply:
I'm going to veto this one and pull it in.

The reason is already above, there have been requests and feedback for it, and was put on the original list by Gabriela for reasons I agree with. I also agree with the concerns Martin raises.

With that said, I think it is easier to remove a rule and clean up docs, than adding a rule and adjusts all docs to include new documentation. 

With that in mind, I've decided to include this rule for now. If it turns out to be cumbersome, not working or problematic, removing it would be fairly easy. Adding it, not so much.

../Frenck